cyences_app_for_splunk/default/data/ui/views/cs_forensics.xml

```diff
@@ -22,9 +22,23 @@
   <row>
     <panel>
       <title>Notable Events</title>
+      <input type="multiselect" token="tkn_severity" searchWhenChanged="true">
+        <label>Severity</label>
+        <choice value="critical">critical</choice>
+        <choice value="high">high</choice>
+        <choice value="medium">medium</choice>
+        <choice value="low">low</choice>
+        <choice value="info">info</choice>
+        <default>critical,high,medium</default>
+        <prefix>(</prefix>
+        <suffix>)</suffix>
+        <valuePrefix>"</valuePrefix>
+        <valueSuffix>"</valueSuffix>
+        <delimiter>,</delimiter>
+      </input>
```

Comment:
We need to add input to show all the alerts including the ones that did not trigger them.
(If Usama doesn't like it, we can just hide it.)
I feel this will be useful in troubleshooting and development, etc.

Just a checkbox for this would suffice.